lms/djangoapps/verify_student/views.py

```diff
@@ -1009,6 +1010,16 @@ def results_callback(request):
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        try:
+            checkpoints = VerificationCheckpoint.objects.filter(photo_verification=attempt)[0]
```

Comment:
Please change `checkpoints` to `checkpoint`. Also why we are getting only one checkpoint?